capeinfra/pipeline/batch.py

```diff
@@ -3,8 +3,9 @@
 import pulumi_aws as aws
 from pulumi import Input, ResourceOptions
 
+from capepulumi import CapeComponentResource
+
 from ..iam import get_inline_role, get_instance_profile
```

Comment:
looks like pretty much everywhere you've gone to fully qualified (vs relative) paths for the imports. i'm totally fine with that. should change here. (and check if i've missed any others)